packages/orbit-components/src/Wizard/WizardStep/index.js

```diff
@@ -0,0 +1,246 @@
+// @flow
+import * as React from "react";
+import { css } from "styled-components";
+import { hex2rgb } from "@csstools/convert-colors";
+
+import useMediaQuery from "../../hooks/useMediaQuery";
+import ButtonLink from "../../ButtonLink";
+import Stack from "../../Stack";
+import Text from "../../Text";
+import CheckCircle from "../../icons/CheckCircle";
+import useTheme from "../../hooks/useTheme";
+
+import type { Props } from ".";
+
+const WizardStep = ({
+  dataTest,
+  number,
+  title,
+  status,
+  nextStepStatus,
+  active,
+  onClick,
+}: Props) => {
+  const { isTablet } = useMediaQuery();
+  const theme = useTheme();
+  const iconWidth = parseFloat(theme.orbit.widthIconSmall);
+  const iconHeight = parseFloat(theme.orbit.heightIconSmall);
+  // account for additional inner spacing because of visual balance
+  const checkboxWidth = iconWidth * (5 / 6);
+  const checkboxHeight = iconHeight * (5 / 6);
+  const colorDisabled = theme.orbit.paletteCloudNormalHover;
+
+  const activeGlowColor = `rgba(${hex2rgb(theme.orbit.paletteProductNormal)
+    .slice(0, -1)
+    .map(n => 255 * (n / 100))
+    .concat(0.2)
+    .join(", ")})`;
+
+  /**
+   * We're explicitly rounding values in layout styles because browsers seem to
+   * be rounding them inconsistently horizontally vs vertically.
+   */
+
+  const icon = (
+    <div
+      css={css`
+        position: relative;
+        opacity: 1;
+        &:before {
+          content: "";
+          display: block;
+          width: ${Math.round(checkboxWidth)}px;
+          height: ${Math.round(checkboxHeight)}px;
+          position: absolute;
+          top: 50%;
+          left: 50%;
+          transform: translate(-50%, -50%);
+          background: ${theme.orbit.paletteProductNormal};
+          ${status === "completed" &&
+          css`
+            background: ${theme.orbit.paletteWhite};
+          `}
+          ${status === "disabled" &&
+          css`
+            background: ${colorDisabled};
+          `}
+          border-radius: 50%;
+          ${active &&
+          css`
+            box-shadow: 0 0 0 4px ${activeGlowColor};
+          `}
+        }
+        svg {
+          position: relative;
+          display: block;
+        }
+      `}
```

Comment:
Just curious. Is there any reason why you don't use SC and pass styles with `css` template literals via css?

Reply:
I'm aware that so far we probably haven't used `css` anywhere, but there are several reasons why I like using it:

- avoiding having to come up with a lot of nonsense component names 😅
- I thinking creating a component for every set of styles is disrupting to the reading flow of the code, I try to keep it "top to bottom" as much as possible
- all that this component does is contain styles, so I see no need to put the styles outside of it because there is no logic to flesh out

I think this is exactly what `css` was designed for, and `styled` might be more suitable for reusable sets of styles, which can be extended into styled new components as needed, etc.